static/js/stats/stats.js

```diff
@@ -311,7 +397,7 @@ function populate_messages_sent_over_time(data) {
 
 $.get({
     url: '/json/analytics/chart_data',
-    data: {chart_name: 'messages_sent_over_time', min_length: '10'},
+    data: {chart_name: 'messages_sent_over_time'},
     idempotent: true,
     success: function (data) {
         populate_messages_sent_over_time(data);
```

Comment:
There is a lot of duplicated code (e.g. layout code, etc) in the current implementation. It will also make it harder to maintain over time. I would suggest instead having pseudocode here of the following form:
```
if (not enough data)
    replace data with fixture data
    make everything opacity .2, disable buttons, etc
    show "not enough data" message
populate_messages_sent_over_time(data);
```